OracleCoherence/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Coherence 12.1.3 Standalone Distribution
+# 
+# REQUIRED BASE IMAGE TO BUILD THIS IMAGE
+# ---------------------------------------
+# Make sure you have oraclelinux:7.0 Docker image installed.
+# Visit for more info: 
+#  - http://public-yum.oracle.com/docker-images/
```

Comment:
Please change this url to https://registry.hub.docker.com/_/oraclelinux/
That's the source of the official Oracle Linux images for Docker.